test/metadata_tests.cpp

```diff
@@ -17,11 +18,17 @@ BOOST_AUTO_TEST_CASE( put_metadata ) {
     const char* bucket_name = "metadata_test";
     FILE* file;
 
-    printf("-----Testing put_metadata-------\n");
+    ds3_request* request = ds3_init_put_bucket_spectra_s3_request(bucket_name);
```

Comment:
You should put the create bucket with data policy calls into a function that can be reused rather than re-typing it in each test.